packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Removes usage of `_ambiguate` method in tests.
```

Comment:
Test-only changes don't need CHANGELOG entries.

Reply:
Thanks, removed it.